src/ui/components/touchableLink/__tests__/TouchableLink.test.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react'
+import { Text } from 'react-native'
+import waitForExpect from 'wait-for-expect'
+
+import { navigate } from '__mocks__/@react-navigation/native'
+import * as NavigationHelpers from 'features/navigation/helpers/openUrl'
+import { navigateFromRef } from 'features/navigation/navigationRef'
+import { homeNavConfig } from 'features/navigation/TabBar/helpers'
+import { mockedFullAddress } from 'libs/address/fixtures/mockedFormatFullAddress'
+import { analytics } from 'libs/firebase/analytics'
+import { getGoogleMapsItineraryUrl } from 'libs/itinerary/openGoogleMapsItinerary'
+import { render, fireEvent } from 'tests/utils'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { SocialNetworkIconsMap } from 'ui/components/socials/types'
+
+import { TouchableLink } from '../TouchableLink'
+
+const openUrl = jest.spyOn(NavigationHelpers, 'openUrl')
+jest.mock('features/navigation/navigationRef')
+
+const navigateToItineraryMock = jest.fn()
+const useItinerary = () => ({
+  navigateTo: navigateToItineraryMock,
+})
+jest.mock('libs/itinerary/useItinerary', () => ({
+  useItinerary,
+}))
+
+const linkText = 'linkText'
+const TouchableLinkContent = () => <Text>{linkText}</Text>
+describe('<TouchableLink />', () => {
+  describe('Internal Navigation', () => {
+    it('should navigate to right screen with expected params (nominal case)', async () => {
+      const { getByText } = render(
+        <TouchableLink navigateTo={{ screen: homeNavConfig[0], params: homeNavConfig[1] }}>
+          <TouchableLinkContent />
+        </TouchableLink>
+      )
+      fireEvent.press(getByText(linkText))
+      await waitForExpect(() => {
+        expect(navigate).toBeCalledWith(...homeNavConfig)
```

Comment:
je trouverais ça plus explicite d'avoir la liste des arguments écrit en dur dans le test

```suggestion
        <TouchableLink navigateTo={{ screen: 'TabNavigator', params: { screen: 'Home', params: undefined } }}>
          <TouchableLinkContent />
        </TouchableLink>
      )
      fireEvent.press(getByText(linkText))
      await waitForExpect(() => {
        expect(navigate).toBeCalledWith('TabNavigator', { screen: 'Home', params: undefined }])
```
de cette manière, pour comprendre le test, je n'ai pas besoin d'aller voir les définition